src/compiler/diagnosticMessages.json

```diff
@@ -1775,6 +1775,14 @@
         "category": "Error",
         "code": 2661
     },
+    "Cannot find name '{0}'. Did you mean to prefix the static member with the class name, '{1}.{0}'?": {

```

Comment:
Maybe @sandersn and @JsonFreeman can weigh in on something better. I prefer my suggestion because it's shorter and less overwhelming for the user.